Makefile

```diff
@@ -499,6 +499,7 @@ build-chart: $(HELM) $(KUSTOMIZE) $(RELEASE_DIR) $(CHART_RELEASE_DIR) $(CHART_PA
 	sed -i'' -e 's@imageVersion: .*@imageVersion: '"$(CONTROLLER_IMAGE_VERSION)"'@' $(CHART_RELEASE_DIR)/values.yaml
 	sed -i'' -e 's@imagePullPolicy: .*@imagePullPolicy: '"$(PULL_POLICY)"'@' $(CHART_RELEASE_DIR)/values.yaml
 	cd $(CHART_RELEASE_DIR) && $(HELM) dependency update
+	cd $(CHART_DIR) && $(HELM) dependency update
```

Comment:
This is unnecessary, we update `Chart.lock` when a new release tag is cut, and `gh-pages` contains latest version of the file. During e2e execution the chart is built and the lock file used is always in `out/charts/ran
cher-turtles/Chart.lock` and it is what the branch contains ATM.

Reply:
I am not understanding how can this be unnecessary and why it should not be always up-to-date, it is making sure this file is up-to-date with chart.yaml and visible in the branch and the `out` folder is gitignored anyways making it confusing

Reply:
There are 2 separate things I’m talking about.
1. Release - packaged in gh-pages. You can see version 0.8.1 there in the lock.
1. Out folder is for tests. You can see 0.9.0 with your PR without change on this line.
Anyways, this is just no-op, noticed and commented. Does not need to be acted upon.

I’m trying to propose stop doing manual bumps, and relying 100% on dependabot for such modifications. This PR is a dependabot responsibility. The only reason why it does not work yet, is because if you bump CAPI Operator version in `/`, `/test` has to be also updated. This creates a diff, and a PR check is failing. The solution I wanted to try (mentioned today already) is adding [this](https://github.com/kubernetes-sigs/cluster-api-addon-provider-helm/blob/main/.github/workflows/dependabot.yml) to give the bot more power.